index.html

```diff
@@ -9700,6 +9700,7 @@ <h5>Presentational Role Inheritance</h5>
 				<p>A <rref>widget</rref> that allows the user to select one or more items from a hierarchically organized collection.</p>
 				<p>To be <a>keyboard accessible</a>, authors SHOULD manage focus of descendants for all instances of this <a>role</a>, as described in <a href="#managingfocus">Managing Focus</a>.</p>
 				<p>Elements with the role <code>tree</code> have an implicit <pref>aria-orientation</pref> value of <code>vertical</code>.</p>
+				<p>Authors MUST ensure that all [=ARIA/owned=] [=elements=] with a <rref>widget</rref> <a>role</a> have either role <rref>treeitem</rref> or role <rref>group</rref>.</p>
```

Comment:
a `group` is not a widget role though?

is this trying to say that a tree may contain elements with a `treeitem` role, or elements with a `group` role, so long as the `group` contains elements with a `treeitem` role, and that only other element must have a generic or presentation role, with no other properties important to accessibility?   e.g., `<div tabindex=lol>`

Reply:
> a group is not a widget role though?

Thanks for catching that. I keep thinking they are but you are of course right.

> is this trying to say that a tree may contain elements with a treeitem role, or elements with a group role, so long as the group contains elements with a treeitem role, and that only other element must have a generic or presentation role, with no other properties important to accessibility?

That's not what I was going for. I understand the point but I'm not sure I'd personally make it that strong. treeitem allows name from content so I'd expect more than just "flat" content (at the very least images but I could see more complex content at the leafs of the tree).

Perhaps we should clarify this in the group.

Reply:
Yeh agreed. And that’s a good idea to define what’s allowed in a group in the group role section. Otherwise we’d need to duplicate whatever’s called out here in other role defs

Reply:
Please rephrase to avoid using the abstract role `widget`? If this is referring to any subrole of widget, expand that list.

Reply:
Coming back to this I'm wondering: isn't #1440 trying to enable some uses cases that this would prevent (e.g., button inside a treeitem as secondary action)?